src/main/java/org/folio/rest/impl/PostGobiOrdersHelper.java

```diff
@@ -145,7 +152,7 @@ public PostGobiOrdersHelper(HttpClientInterface httpClient,
   public CompletableFuture<String> lookupLocationId(String location) {
     try {
       String query = HelperUtils.encodeValue(String.format(CQL_CODE_STRING_FMT, location));
-      return httpClient.request("/locations?query=" + query, okapiHeaders)
+      return httpClient.request("/location?query=" + query, okapiHeaders)
```

Comment:
you're going to laugh (or maybe not), but in this case we actually want to call the inventory-storage /locations (plural).  The gobi order will come in with something like "main circ. desk" or "springfield library".  Here we're looking up the UUID of that location within FOLIO.  The orders-storage /location endpoint is for the "location" part of the poLine, which contains among other things the UUID we look up here.

Reply:
uhh... (yea I did smirk)